src/log/Log.cc

```diff
@@ -376,8 +373,6 @@ void Log::dump_recent()
 
   _log_message("--- end dump of recent events ---", true);
 
-  _flush_logbuf();
```

Comment:
instead of removing `_flush_logbuf()`, could you replace it with `assert(m_log_buf.empty())`?